tests/unit/Middleware/RecordTransactionTest.php

```diff
@@ -47,12 +47,15 @@ protected function _before()
         $this->response->headers = new ResponseHeaderBag();
     }
 
-    protected function createMiddlewareInstance(bool $use_route_uri): void
+    protected function createMiddlewareInstance(bool $use_route_uri, ?string $ignore_patterns): void
```

Comment:
Is this a deliberate choice over setting a default value of `''`? Default value would spare us from changing `createMiddlewareInstance` invocations.

Reply:
Not deliberate. Fixed.